anabot/runtime/installation/hub/partitioning/advanced/__init__.py

```diff
@@ -96,7 +96,12 @@ def schema_manipulate(element, app_node, local_node, check):
         return (False, u"Different schema is set: %s" % schema_node.name)
     # Select new schema
     schema_node.click()
-    getnode(schema_node, "menu item", schema_name(schema)).click()
+    try:
+        getnode(schema_node, "menu item", schema_name(schema)).click()
+    except TimeoutError:
+        # If schema name is valid but not found in the dropdown menu (e.g.
```

Comment:
Oh, I can see that this statement was not right. It's true that in case of other "invalid" schema name, the TimeoutError would not be raised as it would be KeyError, my bad, I should have checked the behaviour of the `schema_name` function.